src/plugins/BunqLayoutPlugin/BunqLayoutPlugin.js

```diff
@@ -0,0 +1,173 @@
+// @flow
+import BunqLayout from "./components/BunqLayout/BunqLayout";
+import Sidebar from "./components/Sidebar/Sidebar";
+
+/**
+ * @type {Object}
+ */
+const actions = {
+    /**
+     * @param {Object} apiSpecification
+     *
+     * @returns {Object}
+     */
+    updateSidebarEndpoints: (apiSpecification: Object): Object => {
+        const tags = [];
+
+        if (apiSpecification.tags) {
+            for (const tag of apiSpecification.tags) {
+                tags.push(tag.name);
+            }
+        }
+
+        return {
+            type: "BUNQ_UPDATE_SIDEBAR_ENDPOINTS",
+            payload: tags
+        }
+    },
+
+    /**
+     * @param {Function} originalAction
```

Comment:
it would work, but since `function` is usually highlighted by IDEs I'd keep it as `Function` to avoid any confusion when using flow types.

![image](https://user-images.githubusercontent.com/4534250/47900817-8ac1c380-de7e-11e8-9231-f341c8127728.png)

Reply:
also it throws an error with `function` :D